### PR TITLE
Update mattermost-limits.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Doc update for https://github.com/mattermost/mattermost-webapp/pull/11935

### CODE REVIEW

File: source/onboard/mattermost-limits.rst [hunk 1]
```diff
@@ -20,17 +20,19 @@ Mattermost Free limits
...
-- Unlimited plugins can be installed but only 5 can be enabled at one time. Core plugins such as Boards, Calls, and Playbooks do not count toward this limit.
+- 10 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived. Card attachments are counted toward this limit.
 
 **Channels limits**
 
 - 10,000 message history limit. The oldest messages that exceed this limit will be archived.
 
+**Calls limits**
+
+- Up to 8 participants per group call.
```
Comment (justinegeffen) [inline source/onboard/mattermost-limits.rst:31]: @tao - is this per individual call? Can there be multiple calls of up to 8 people at any given time?

File: source/onboard/mattermost-limits.rst [hunk 2]
```diff
@@ -20,17 +20,19 @@ Mattermost Free limits
...
+
 **Boards limits**
 
 - Maximum 5 saved views per board.
```
Comment (wuwinson) [inline source/onboard/mattermost-limits.rst:35]:
```suggestion
```

File: source/onboard/mattermost-limits.rst [hunk 3]
```diff
@@ -20,17 +20,19 @@ Mattermost Free limits
...
 
 **Channels limits**
 
 - 10,000 message history limit. The oldest messages that exceed this limit will be archived.
 
+**Calls limits**
+
+- Up to 8 participants per group call.
+
 **Boards limits**
```
Comment (wuwinson) [inline source/onboard/mattermost-limits.rst:33]:
```suggestion
```

**Review by wuwinson — APPROVED** [review]
All Boards limits will be removed.

File: source/onboard/mattermost-limits.rst [hunk 4]
```diff
@@ -19,17 +19,17 @@ Mattermost Cloud Free limits
...
+
 
 **Channels limits**
 
 - 10,000 message history limit. The oldest messages that exceed this limit will be archived.
 
 **Calls limits**
 
-- Up to 8 participants per channel can join calls that last up to 40 minutes.
+- Up to 8 participants per channel
```
Comment (justinegeffen) [inline source/onboard/mattermost-limits.rst:32]: @itao, do we want to mention the 200 limit in this doc as well?

**Comment by justinegeffen** [comment]
@asatkinson, @itao: gentle reminder on this PR, thanks!

**Review by asatkinson — APPROVED** [review]
approving removing limits on # of plugins/apps.

**Pull request merged**